dbdeployer.hcl

```diff
@@ -0,0 +1,16 @@
+description = "DBdeployer is a tool that deploys MySQL database servers easily.."
+binaries = ["dbdeployer"]
+test = "dbdeployer --version"
+strip = 1
+source = "https://github.com/datacharmer/dbdeployer/releases/download/v${version}/dbdeployer-${version}.${os}.tar.gz"
```

Comment:
The build is failing because `darwin` variant of this download URL doesn't exist. Take a look at [this guide](https://cashapp.github.io/hermit/packaging/tutorial/?h=os#add-download-urls-for-each-os) for how to add download URLs for each platform.